GoogleForCreators/web-stories-wp
# Dashboard filter by taxonomy

## Context


Add filter by story category to the dashboard.

## Summary


Give the user the ability to filter by story category.  Similar to the author filter.

## Relevant Technical Choices


Mirrors author filter on the dashboard.

## To-do



## User-facing changes


before/after
https://user-images.githubusercontent.com/14057928/169389100-4fc7bd45-1174-4b4b-9640-0aa0e0a795e9.mp4

## Testing Instructions






This PR can be tested by following these steps:

1. Go to story dashboard

On Staging:
- Notice there is no 'Category' filter

On Branch:
- Notice there is a 'Category' filter
2. Make sure there are stories with 'categories' assigned to them
3. Test filter correctly filters stories
4. Test that the search within the filter dropdown is working correctly as well (to search for category to filter by)


## Reviews

### Does this PR have a security-related impact?
No


### Does this PR change what data or activity we track or use?
No


### Does this PR have a legal-related impact?
No


## Checklist




---



Fixes #11436
